tests/dogtag/pytest-ansible/installation/roles/Test_Execution/tasks/configure_common.yml

```diff
@@ -2,44 +2,15 @@
   systemd: state=stopped name=firewalld
   when: ansible_distribution == "RedHat"
 
-- name : Set hostname for machines Bydefault we choose pki1 for master and pki2 for clones.
```

Comment:
These steps are executed in gitlab pipeline run. No issues in deleting this, if we are planning to execute all the tests here in github.
But if not, we can not remove these.

Reply:
This test will be executed automatically on GitHub as part of upstream CI test on every checkin. I think the question is whether QE will consider this as a valid QE test, or does it have to be run in GitLab pipeline in order to be valid?

Another option is to change the GitLab environment such that it doesn't require changing the hostname, if that's possible. Or maybe add rules such that the hostname changes are only done in GitLab. I'm not familiar with GitLab rules, so if you could provide that I can add it into the PR.

Reply:
Just to clarify, these commands failed on Docker because some files are readonly in the container, so it has to be removed/skipped. The hostname is set when the container is created, before running Ansible playbook.

Reply:
@06shalini I added `ansible_connection != "docker"` as a condition to run these steps. The `ansible_connection` is set to `docker` in the inventory file. Will GitLab use a different inventory file?